### PR TITLE
[OA][OPS-9418] Accessibility job failure fixed

### PR DESCRIPTION
**Pull request by Akure4Life**
Signed-off-by: olabode <15871275+Akure4Life@users.noreply.github.com>

### CODE REVIEW

File: app/views/calculator/payment_summary.scala.html [hunk 1]
```diff
@@ -78,8 +78,8 @@ <h1 class="govuk-heading-xl">@messages("ssttp.calculator.form.payment_summary.ti
             @messages("ssttp.calculator.form.payment_summary.amount.remaining")
           </dt>
           <dd class="govuk-summary-list__value">
-            @currency(debits.map(_.amount).sum - initialPayment)
-              <p class="small govuk-!-font-weight-regular govuk-!-margin-top-1">@messages("ssttp.calculator.form.payment_summary.amount.remaining.hint")</p>
```
Comment (Kntanos) [inline app/views/calculator/payment_summary.scala.html:82]: Is there a reason for not having a 'p' element here and in line 63?

File: app/views/core/you_need_to_file.scala.html [hunk 2]
```diff
@@ -51,7 +51,7 @@ <h1 class="govuk-heading-xl">@messages("ssttp.you-need-to-file.title")</h1>
         </div>
     </details>
     <section class="govuk-body">
-        <h3>@messages("ssttp.call-us.opening-times.heading")</h3>
+        <h2>@messages("ssttp.call-us.opening-times.heading")</h2>
```
Comment (Kntanos) [inline app/views/core/you_need_to_file.scala.html:54]: It looks like in all other eligibility kick out pages the 'Opening times' line is not a h2, but normal text. Should we change it to h2 or keep it consistent with the other pages?

File: app/views/calculator/payment_summary.scala.html (hunk 1 above)
Comment (Akure4Life) [reply app/views/calculator/payment_summary.scala.html:82]: Jenkins was reporting it as an error in those 2 places.

File: app/views/core/you_need_to_file.scala.html (hunk 2 above)
Comment (Akure4Life) [reply app/views/core/you_need_to_file.scala.html:54]: The other pages don't have H2 or H3 tags

**Review by Kntanos — APPROVED** [review]

**Pull request merged**